chia/plotting/plot_tools.py

```diff
@@ -316,3 +316,41 @@ def find_duplicate_plot_IDs(all_filenames=None) -> None:
         for filename_str in duplicate_filenames:
             log_message += "\t" + filename_str + "\n"
         log.warning(f"{log_message}")
+
+
+def load_plot(
+    filename: str,
+) ->  Tuple[PlotInfo,str]:
+    if filename==None:
+        return  {},"use -f to add a plot file, use -h to see usage"
+   
+    filename=Path(filename)
+    if filename.exists():
+        try:
+            prover = DiskProver(str(filename))
+
+            expected_size = _expected_plot_size(prover.get_size()) * UI_ACTUAL_SPACE_CONSTANT_FACTOR
+            stat_info = filename.stat()
+
+            if prover.get_size() >= 30 and stat_info.st_size < 0.98 * expected_size:
+                return {}, f"Not farming plot {filename}. Size is {stat_info.st_size / (1024**3)} GiB, but expected,\nat least: {expected_size / (1024 ** 3)} GiB. We assume the file is being copied."
```

Comment:
this looks like a copy-pasted error message. We're asking for information about a plot, not whether we're farming it or not.

Reply:
Thank you for your guidance, I learned new knowledge. The format issues have been fixed.